configuration/white_labels/nc.py

```diff
@@ -20,24 +20,15 @@ def get_white_label(self) -> WhiteLabel:
     more_help_options = {
         "moreHelpOptions": [
             {
-                "name": {
-                    "_default_message": "2-1-1 North Carolina",
-                    "_label": "moreHelp.resource_name1",
-                },
+                "name": {"_default_message": "2-1-1 North Carolina", "_label": "moreHelp.nc_resource_name1"},
                 "link": "https://nc211.org/",
-                "phone": {
-                    "_default_message": "Dial 2-1-1 or 866.760.6489",
-                    "_label": "moreHelp.resource_phone1",
-                },
+                "phone": {"_default_message": "Dial 2-1-1 or 866.760.6489", "_label": "moreHelp.resource_phone1"},
```

Comment:
@Linda-Vu wondering if we should change this label to `moreHelp.nc_resource_phone1` so that we can change the area code?

Reply:
@kbrey nice catch!! let's update that config to be

```
      "phone": {
        "_default_message": "Dial 2-1-1 or 1-888-892-1162",
        "_label": "moreHelp.nc_resource_phone1"
      }
```

Reply:
Done!